docs/runtime_api.md

```diff
@@ -579,24 +579,17 @@ sendExample(new Example());
 their methods. 
 
 In contrast to `PartialMessage`, `PlainMessage` requires all properties to be
-provided. And since it is not recursive, message fields must provide the exact type.
-For example:
+provided. For example:
 
 ```typescript
 let plain: PlainMessage<Example> = {
   foo: "abc",
   bar: false,
-  baz: undefined,
+  baz: 42,
```

Comment:
`baz: 42` is a type error. At the top, we define it as a message field. Feel free to remove the line entirely if you think `baz: undefined` is unclear.